src/light_data/light_store.rs

```diff
@@ -137,30 +141,49 @@ impl<F: LurkField> LightStore<F> {
     fn to_scalar_store(&self) -> anyhow::Result<ScalarStore<F>> {
```

Comment:
This will probably require adapting `test_convert_light_store_basic_symbols`

Reply:
But actually the test looks like it's different on Syms with 1 char strings, I don't know why what I did would change that?